examples/time-travel.ipynb

```diff
@@ -46,7 +46,10 @@
    "id": "af4ce0ba-7596-4e5f-8bf8-0b0bd6e62833",
```

Comment:
why did this change?

Reply:
typo. had to change from ANTHROPIC to OPENAI